scripts/publish.py

```diff
@@ -48,7 +48,7 @@
         "wasmer-cli": "default,cranelift",
         "wasmer-wasix": "sys,wasmer/sys",
         "wasmer-wasix-types": "wasmer/sys",
-        "wasmer-wasix-experimental-io-devices": "wasmer-wasix/sys",
+        "wasmer-wasix-experimental-io-devices": "wasmer-wasix/sys,wasmer/sys",
```

Comment:
This is the only one with multiple features, does the code properly handle splitting? 
Or is this just passed to `-F`  or similar so it works fine?

Reply:
It's not the only one. `wasmer-cli` and `wasmer-wasix` also have 2 features. the while features test is used as-is with no issues.